src/overview.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+## How to Read the Reader?
+
+First off, this reader is written for Roux users of all levels, as long as you know beginner Roux. (If not, check out this awesome beginner tutorial [guide](https://tutorial.rouxers.com) here).
+
+There isn't a single recommended way of reading it, but here are some useful scenarios:
+
+- If you already have a topic in mind (e.g. FB turning, EOLR progression), go find the related sections, and read the explanations to help you select the best resources.
+- If you are just randomly looking for things to learn, skim through the book for whatever interests you, and read up the section overview to discover exciting skills, techniques or concepts to work on.
+
+- If you have a particular question in mind, it might have been asked by many before. That's why we have created the [FAQ chapter](book.rouxers.com/faqs.html). Try your luck there!
+
+## Organization
+
+At the top level, the reader is organized by the four steps in Roux -- FB, SB, CMLL & LSE, plus a few chapters for generic topics that don't belong to a step. In Blockbuilding, we go over the three fundamental aspects of speedcubing -- **solutions, fingertricks, and lookahead**. For each of these subsections, we first provide an **overview** of what a fast cuber should be ideally doing, and talk about the **progression** -- what do learn at different skill levels. This overview is followed by a curated list of **resources** with commentaries. In CMLL and LSE, this structure is generally followed, with the additional emphasis on sub-step variants and their algorithms. Lastly, while OH and 2H techniques can differ sometimes, they are usually discussed within the same subsections to avoid fragmenting.
+
+## How to Contribute
+
+If you wish to contribute to the writing, feel free to contact any one of the authors. Please refer to the organization section above when making edits.
+A basic knowledge of `github` and running simple commands from the terminal would be helpful. The workflow is simple enough: Basically, download the binary for [mdbook](https://github.com/rust-lang/mdBook/releases/), clone this repo and make edits to the Markdown files locally, generate the book by running `mdbook serve` on the command line, and submit a pull request when you're done! The book will be automatically generated using a CI workflow once new changes are checked in.
```

Comment:
instead of `github`, maybe [GitHub](https://github.com/rouxers/book) ?

Reply:
lol github but it links to the book